package/AUTHORS

```diff
@@ -190,7 +190,7 @@ Chronological list of authors
   - Aditi Tripathi
   - Sukeerti T
   - Robot Jelly
-
```

Comment:
Don't remove this blank line please. Once that is done all good to go.

Reply:
Yep I've added it back.